modules/InfiniteStories/InfiniteStories.tsx

```diff
@@ -64,8 +68,8 @@ export function InfiniteStories({
     const locale = useLocale();
     const { load, loading, data, done } = useInfiniteLoading(
         useCallback(
-            (offset) => fetchStories(locale, offset, pageSize, category, excludedStoryUuids),
-            [category, excludedStoryUuids, locale, pageSize],
+            (offset) => fetchStories(locale, offset, pageSize, category, excludedStoryUuids, tag),
```

Comment:
So many arguments already. It probably makes sense to pass these params as an object 🤔

Reply:
https://github.com/prezly/theme-nextjs-bea/pull/1199/commits/d08d648725706d30a8f48e0e3272e04e2b1ed7a0